src/events.ts

```diff
@@ -89,11 +89,6 @@ export type NoiseDetectedEvent =
 export type NoiseThresholdTriggeredEvent = CommonDeviceEvent<
   "noise_sensor.noise_threshold_triggered",
   {
-    noise_threshold_id: string
-    noise_threshold_name: string
-    noise_level_decibels: number
-
-    noise_level_nrs?: number
```

Comment:
should we put the noise_threshold_id in here, seems weird not to provide at least noise_level_decibels?

Reply:
i feel like this payload is actually ok, why remove it?

Reply:
Context https://github.com/seamapi/seam-connect/pull/2978#discussion_r1194216123

Reply:
ok understood- should we make optional instead of removing

Reply:
fwiw i'm behind on context so whatever you guys decide is cool 😁 sounds like a temporary thing

Reply:
Removing is safer if nothing actually uses it yet IMO. Yes we will have to add it back here later, but it won't confuse customers this way when they try to use there and it's always null.

Reply:
👍

Reply:
jsdocs can also be a good way to do indicate it's not populated while still giving an impression of the eventual intent

Reply:
Side note, removing or making something nullable is always a breaking change for types like this. But I am assuming this was not part of the stable api yet?

Reply:
nobody has built on the typescript sdk for this feature yet afaik, but i don't think we're doing a great job of tracking @itelo it might make sense to have a simple admin page for seeing all the noise sensors in prod or tracking usage. I know Ravi has a bunch of little admin pages for various monitoring operations- that could help inform if this change would be breaking